openassessment/xblock/static/js/spec/studio/oa_edit.js

```diff
@@ -47,7 +47,7 @@ describe("OpenAssessment.StudioView", function() {
     var EXPECTED_SERVER_DATA = {
         title: "The most important of all questions.",
         prompts: [{"description": "How much do you like waffles?"}, {description : 'How much do you like waffles 2?'}],
-        feedbackPrompt: "",
```

Comment:
nothing for feedback_default_text?

Reply:
Whoops, the test wasn't originally there for feedback_default_text. Just added.